Frameworks/Foundation/MainDispatcher.mm

```diff
@@ -85,6 +85,34 @@ void _DispatchMainRunLoopOnUIThread(int signaledEvent) {
 }
 
 /**
+* Private method to create the dispatcher
+*/
+void _EnsureDispatcher() {
+    if (s_dispatcher == nullptr) {
```

Comment:
Good point.  This is always initialized in the main thread, so the _EnsureDispatcher in the other thread is unnecessary, I'll remove that.

---
In reply to: [83293021](https://github.com/Microsoft/WinObjC/pull/1142#discussion_r83293021) [](ancestors = 83293021)